ci_framework/roles/libvirt_manager/tasks/deploy_edpm_compute.yml

```diff
@@ -45,7 +45,8 @@
           'EDPM_COMPUTE_RAM': compute_config.memory,
           'EDPM_COMPUTE_DISK_SIZE': compute_config.disksize,
           'CRC_POOL': compute_config.image_local_dir,
-          'DISK_FILEPATH': cifmw_libvirt_manager_basedir + "/workload/edpm-compute-" + item + ".qcow2"
+          'DISK_FILEPATH': cifmw_libvirt_manager_basedir + "/workload/edpm-compute-" + item + ".qcow2",
+          'CENTOS_9_STREAM_URL': compute_config.image_url | default(cifmw_discovered_image_url)
```

Comment:
Where is this new `CENTOS_9_STREAM_URL` used?

```
[dasm@fedora ci-framework]$ grep -rn CENTOS_9_STREAM_URL *
[dasm@fedora ci-framework]$ 
```

Reply:
Inside install_yamls :) Is not in out repo

Reply:
ack